packages/rest/src/router/swagger-route.ts

```diff
@@ -0,0 +1,44 @@
+import {RedirectRoute} from '.';
+import {RequestContext} from '../request-context';
+import {OperationArgs, OperationRetval, Request} from '../types';
+import {RestServerConfig, ApiExplorerOptions} from '../rest.server';
+import {getUrlForClient, getProtocolForRequest} from '../rest-utils';
+
+export class SwaggerRoute extends RedirectRoute {
+  constructor(
+    public readonly sourcePath: string,
+    public config: RestServerConfig,
+    public statusCode: number = 303,
+  ) {
+    super(sourcePath, '', statusCode);
+    this.config = config;
+  }
+
+  async invokeHandler(
+    {request, response}: RequestContext,
+    args: OperationArgs,
+  ): Promise<OperationRetval> {
+    const config = this.config.apiExplorer!;
+    if (config.disabled) {
+      response.statusCode = 404;
+      response.send();
+      return;
+    }
+    this.targetPath = this.extarctFullUri(request, config);
+    // https://github.com/strongloop/loopback-next/issues/1623
+    response = response.set('location', this.targetPath);
+    response.redirect(this.statusCode, this.targetPath);
```

Comment:
`response.redirect` sets up `Location` header, doesn't it?

Reply:
out of the blue no ! i was expect to be included it seems express was doing the job .

Reply:
I confirm that Express does the job - https://github.com/expressjs/express/blob/master/lib/response.js#L922

Reply:
using express middileware was set the header location, it's doesn't appear since using the route.
perhaps there something  wrong in LB4 Routing ,  reset the headers ?

Reply:
@raymondfeng  
anyway removing this line , will redirect the request without location header,

Reply:
We should have a test to check the `location` header to ensure it's set correctly.

Reply:
@raymondfeng 
we already have more than one. that check the header location content. 
 https://github.com/strongloop/loopback-next/blob/4d37ff1d1757d4d0fa3677a069c4fbaca051803f/packages/rest/test/integration/rest.server.integration.ts#L459 
https://github.com/strongloop/loopback-next/blob/4d37ff1d1757d4d0fa3677a069c4fbaca051803f/packages/rest/test/integration/rest.server.integration.ts#L493
https://github.com/strongloop/loopback-next/blob/4d37ff1d1757d4d0fa3677a069c4fbaca051803f/packages/rest/test/integration/rest.server.integration.ts#L512
https://github.com/strongloop/loopback-next/blob/4d37ff1d1757d4d0fa3677a069c4fbaca051803f/packages/rest/test/integration/rest.server.integration.ts#L652
etc ..

Reply:
not relevant after rolling back